specification/instr/ldga.tex

```diff
@@ -6,28 +6,31 @@ \subsection*{LDGA: load a DTrace built-in variable}
 
 \subsubsection*{Format}
 
-\textrm{LDGA \%rd, \%r1, \%r2}
+\textrm{LDGA \%rd, var, ndx}
 
 \begin{center}
 \begin{bytefield}[endianness=big,bitformatting=\scriptsize]{32}
-\bitheader{0,7,8,15,16,23,24,31} \\
-\bitbox{8}{0x28}
-\bitbox{8}{r1}
-\bitbox{8}{r2}
+\bitheader{0,7,8,15,16,23,24,31}\\
+\bitbox{8}{op}
+\bitbox{8}{var}
+\bitbox{8}{ndx}
 \bitbox{8}{rd}
 \end{bytefield}
 \end{center}
 
 \subsubsection*{Description}
 
 The \instruction{ldga} instruction looks up the value of a DTrace
-built-in variable based on the value in \registerop{r1} with an
-optional array index in \registerop{r2}.
+built-in variable based on the value in \registerop{var} with an
+optional array index in \registerop{ndx}. \\
```

Comment:
Makes the pdf look more legible IMO. Can remove if you think it shouldn't be there (same idea is in ldgs).